proto/cosmos/consensus/v1/tx.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+package cosmos.consensus.v1;
+
+import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
+import "cosmos/msg/v1/msg.proto";
+import "tendermint/types/params.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/consensus/types";
+
+// Msg defines the bank Msg service.
+service Msg {
+  // UpdateParams defines a governance operation for updating the x/consensus_param module parameters.
+  // The authority is defined in the keeper.
+  //
+  // Since: cosmos-sdk 0.47
+  rpc UpdateParams(MsgUpdateParams) returns (MsgUpdateParamsResponse);
+}
+
+// MsgUpdateParams is the Msg/UpdateParams request type.
+//
+// Since: cosmos-sdk 0.47
+message MsgUpdateParams {
+  option (cosmos.msg.v1.signer) = "authority";
+
+  // authority is the address of the governance account.
+  string authority = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+
+  // params defines the x/consensus_params parameters to update.
+  //
+  // NOTE: All parameters must be supplied.
+  tendermint.types.BlockParams     block     = 2 [(gogoproto.nullable) = false];
```

Comment:
Any reason we keep gogoproto annotations?

I don't think they're mandatory for this PR to work, and it's more work when we switch to puslar

Reply:
removed, don't think its needed